mautic/mautic
# Fix not displaying error information when an ajax call goes wrong

… there is responseText



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



In [PR #11852](https://github.com/mautic/mautic/pull/11852) I added the flash message when an ajax call has an error and stopped the display of the error data. This PR fixes it.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check whether the flash message is displayed when something goes wrong:
    - Open the import contact page. The max amount to upload is here. 
    - Try uploading a larger file. The flash message is displayed.
3. Check whether the error information is displayed. The error case should be used.
